caffe2/onnx/onnxifi_transformer.cc

```diff
@@ -0,0 +1,371 @@
+#include "caffe2/onnx/onnxifi_transformer.h"
+
+#include <iostream>
+#include <unordered_set>
+
+#include <google/protobuf/text_format.h>
+
+#include "caffe2/core/context.h"
+#include "caffe2/core/logging.h"
+#include "caffe2/core/operator.h"
+#include "caffe2/core/tensor.h"
+#include "caffe2/onnx/onnx_exporter.h"
+#include "caffe2/opt/backend_cutting.h"
+
+namespace caffe2 {
+
+namespace {
+
+// TODO(yinghai): Remove the awkward conversion between unordered_map and map
+std::unordered_map<std::string, TensorShape> InferShapes(
+    Workspace* ws,
+    NetDef* pred_net,
+    CaffeMap<std::string, TensorShape>* shape_hints_ordered) {
+  // Populate shapes from workplace
+  const std::vector<string>& ws_blobs = ws->Blobs();
```

Comment:
Do you import `std::` namespace somewhere? Probably this should be `std::vector<std::string>`